app/src/main/java/com/keylesspalace/tusky/TabPreferenceActivity.kt

```diff
@@ -74,6 +74,12 @@ class TabPreferenceActivity : BaseActivity(), Injectable, ItemInteractionListene
 
     private val hashtagRegex by lazy { Pattern.compile("([\\w_]*[\\p{Alpha}_][\\w_]*)", Pattern.CASE_INSENSITIVE) }
 
+    private val onFabDismissedCallback = object: OnBackPressedCallback(false) {
```

Comment:
why `false`?

Reply:
So it is inactive. It gets activated when the FAB is extended and on back press collapses it. When the FAB is not extended I want the default behavior on back press = close the activity.